playbooks/roles/mysql/tasks/main.yml

```diff
@@ -21,4 +21,12 @@
   when: remove_experimental_mysql
 
 - include: mysql.yml
-  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
\ No newline at end of file
+  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
+
+- name: Create remote root user
```

Comment:
@mrgnr Why do we need an external root user?

Insights only requires a read_only user, the same that edxapp uses.

Maybe we need it for some other reason.